docs/source/quantization.md

```diff
@@ -143,14 +143,13 @@ Operator coverage varies between dynamic and static quantization and is captured
 |                           |                   | computations       |
 |                           |                   | stay in fp32       |
 +---------------------------+-------------------+--------------------+
+```
 
+### Eager Mode Quantization
 
-Eager Mode Quantization
-^^^^^^^^^^^^^^^^^^^^^^^
-For a general introduction to the quantization flow, including different types of quantization, please take a look at `General Quantization Flow`_.
+For a general introduction to the quantization flow, including different types of quantization, please take a look at {ref}`General Quantization Flow`.
```

Comment:
Best to fix this with specifying a target:
```suggestion
For a general introduction to the quantization flow, including different types of quantization, please take a look at {ref}`General Quantization Flow <general-quantization-flow>`.
```
And add the target as shown below on line 686. Do this for all [these errors](https://github.com/pytorch/pytorch/actions/runs/15599673801/job/43942469844?pr=155520#step:11:977).